### PR TITLE
SAK-41039 - Google Analytics: Add support to anonymize the IP

### PR DESCRIPTION
**Pull request by mpellicer**

### CODE REVIEW

File: config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties [hunk 1]
```diff
@@ -88,6 +88,11 @@
 #
 # portal.google.universal_analytics_id=UA-XXXXXX-XX
 
+# SAK-41039
+# Support to anonymize IP in Google Analytics
```
Comment (buckett) [inline config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Worth mentioning that it only works with Universal Google Analytics?

Comment (mpellicer) [reply config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties]: Yes I guess, the patch comes from the DEV list and it's inside the universal GA if.

**Comment by mpellicer** [comment]
Let's wait for the feedback of Sanghyun Jeon about this PR.

**Comment by csev** [comment]
Good idea.

**Pull request merged**